### DISCUSSION TITLE
How to prevent note-tree from acting like draw menu when I click some of it's child notes

### DISCUSSION THREAD

**Discussion by MotikaCraft-TheJoshua** [TriliumNext/Trilium#4361] [Q&A]
I hope this GIF explains what i want to ask much much better than my question, very sorry if it sounds confusing.

Basically, is there a way to prevent note-tree dropdown that happens when i click any of its child notes?

![brave_qezBGj0GmY](https://github.com/zadam/trilium/assets/56804505/fedab19e-5fff-42c0-9381-e4051510e240)

Thank you in advance!

**Comment by zadam** [comment]
Well, no. The tree pane and note pane are in sync. The tree pane will always show the note opened in the right pane ...
